app/src/main/java/org/mozilla/tv/firefox/onboarding/ReceiveTabPreboardingActivity.kt

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.onboarding
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+import android.preference.PreferenceManager
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.receive_tab_preboarding.buttonNotNow
+import kotlinx.android.synthetic.main.receive_tab_preboarding.buttonSignIn
+import kotlinx.android.synthetic.main.receive_tab_preboarding.descriptionText
+import org.mozilla.tv.firefox.MainActivity
+import org.mozilla.tv.firefox.R
+import org.mozilla.tv.firefox.ext.serviceLocator
+
+class ReceiveTabPreboardingActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.receive_tab_preboarding)
+
+        descriptionText.text = resources.getString(
+            R.string.receive_tab_onboarding_description,
+            resources.getString(R.string.firefox_tv_brand_name_short),
+            resources.getString(R.string.firefox_tv_brand_name)
+        )
+
+        buttonSignIn.setOnClickListener {
+            application.registerActivityLifecycleCallbacks(object : Application.ActivityLifecycleCallbacks {
```

Comment:
nit: can we make an `ActivityLifecycleCallbacksAdapter` with default implementations to simplify this call site? I wonder if anything already exists, e.g. in ktx, for this.

Reply:
Looks like we're changing this approach, so resolving this comment.